frontend/src/Activity/Queue/QueueRow.js

```diff
@@ -44,14 +44,15 @@ class QueueRow extends Component {
     this.setState({ isRemoveQueueItemModalOpen: true });
   };
 
-  onRemoveQueueItemModalConfirmed = (blocklist) => {
+  onRemoveQueueItemModalConfirmed = (blocklist, skipredownload) => {
```

Comment:
This differs from Lidarr, but Lidarr's variable should be renamed. `skipRedownload` (several places this should be changed)